### PR TITLE
Change to redis-sentinel

### PR DESCRIPTION
**Pull request by nz-neo**

### CODE REVIEW

File: src/utils/redis.ts [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
-});
+export const redisClient = function() {
+  if (redisSentinels.length > 0) {
+    console.log("Use sentinels:", redisSentinelName, redisSentinels);
+    return new Redis({
+      sentinels: redisSentinels,
+      name: redisSentinelName,
+    });
+  } else {
+    console.log("Use redis:", redisUrl);
```
Comment (HZHAIX) [inline src/utils/redis.ts:15]: [suggestion] use Logger: `export const redisLogger = getLogger('redis');` other than `console.log`

**Review by HZHAIX — APPROVED** [review]

**Comment by HZHAIX** [comment]
Just small suggestion, LGTM

File: src/utils/redis.ts (hunk 1 above)
Comment (nz-neo) [reply src/utils/redis.ts:15]: If use `redisLogger.info('Use sentinels:', redisSentinelName, redisSentinels);` it not work. how to write it ?

Comment (HZHAIX) [reply src/utils/redis.ts:15]: Ah, need to put all the fields in one string, like: 
```
redisLogger.info(`Use sentinels: ${redisSentinelName} ${redisSentinels}`)
```

Comment (nz-neo) [reply src/utils/redis.ts]: Fixed.

**Pull request merged**